code/controllers/subsystem/discord.dm

```diff
@@ -156,7 +156,7 @@ SUBSYSTEM_DEF(discord)
 	// While there's a collision in the token, generate a new one (should rarely happen)
 	while(not_unique)
 		//Column is varchar 100, so we trim just in case someone does us the dirty later
-		one_time_token = trim("[pick(common_words)]-[pick(common_words)]-[pick(common_words)]-[pick(common_words)]-[pick(common_words)]-[pick(common_words)]", 100)
+		one_time_token = trim(uppertext("PLX-VERIFY-[trim(ckey_for, 5)]-[random_string(16, GLOB.hex_characters)]"), 100)
```

Comment:
I would suggest against this, as it makes retyping the token into discord much more tedious.

I'm aware that the intention is for them to select the code, copy, then paste - however, not everyone will think to do that, or even know that you can. That's why the original uses common words - because it's much easier to copy six common words than to repeatedly tab back and forth for a 33-character randomly generated string.

Reply:
well, then I can restructure the window that shows you your code. Hell I'll make it a TGUI window, and make a copy to clipboard command. I really just don't want the possibility of some code being really sexual or gross

Reply:
common words list is used in a lot of places, and i dont think people want me touching it.
regardless, I'm going to keep it like this and pretty it up so people can copy it to clipboard.

Reply:
guh sorry i need to stop being so pushbacky. would you think refactoring to a tgui window and adding a copy to clipboard functionality would work?

Reply:
Perhaps. But I really think the best option would be to just use 6 random common words, because how often will it *really* put out vaguely sexual codes?

Also, relevant here: https://xkcd.com/936/

Reply:
For what it's worth though, if sexual content is still a concern, perhaps we could change to use EFF's Diceware Word List (though this would require reformatting their provided TXT): https://www.eff.org/deeplinks/2016/07/new-wordlists-random-passphrases

Reply:
I think it would work.

That said, I really think the best option would be to just use 5 or 6 random common words, because I don't see sexual content coming up that often. If sexual content is still a concern though, perhaps we could change to use EFF's Diceware Word List (though this would require reformatting their provided TXT): https://www.eff.org/deeplinks/2016/07/new-wordlists-random-passphrases

Aside, here's a relevant XKCD: https://xkcd.com/936/